tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -415,6 +415,20 @@ describe(__filename, () => {
     }));
   });
 
+  it('autofills slug when name is entered while creating collection', () => {
+    const name = "trishul's collection";
+    const localStore = dispatchClientMetadata().store;
+    dispatchSignInActions({
+      store: localStore,
+    });
```

Comment:
Why do you need this actually? If I am not mistaken, it is already initialized like this in a `beforeEach` hook.